src/bokeh/util/sampledata.py

```diff
@@ -56,39 +56,48 @@ def download(progress: bool = True) -> None:
 
     # HTTP requests are cheaper for us, and there is nothing private to protect
     s3 = 'http://sampledata.bokeh.org'
-    files = json.load(open(Path(__file__).parent / "sampledata.json"))
+    files = metadata()
 
-    for filename, md5 in files:
-        real_name, ext = splitext(filename)
-        if ext == '.zip':
-            if not splitext(real_name)[1]:
-                real_name += ".csv"
-        else:
-            real_name += ext
-        real_path = data_dir /real_name
+    for file_name, md5 in files.items():
```

Comment:
suggest 
```
for file_name, md5 in metadata().items():
```
since `files` is not otherwise used and `files = metadata()` reads as confusing